src/components/OpenOrder.js

```diff
@@ -6,29 +6,47 @@ import EntryComponent from './Entry';
 import MasterItemsComponent from './MasterItems';
 import ReadyForBillBtnComponent from './ReadyForBillBtn';
 import { Link as LinkComponent } from 'react-router';
+import FilterComponent from './Filter';
 
 const Entry = createFactory(EntryComponent);
 const MasterItems = createFactory(MasterItemsComponent);
 const ReadyForBillBtn = createFactory(ReadyForBillBtnComponent);
 const Link = createFactory(LinkComponent);
+const Filter = createFactory(FilterComponent);
+const openAndCompleted = `${ constants.OPEN }/${ constants.COMPLETED } `;
```

Comment:
This feels a little weird...but not sure how else to handle something like this.